src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -179,6 +179,27 @@ export default class DefaultAudioVideoController
     this._mediaStreamBroker.addMediaStreamBrokerObserver(this._audioMixController);
     this.meetingSessionContext.logger = this._logger;
     this._eventController = eventController;
+
+    if (!this.configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers) {
```

Comment:
Can we provide more context regarding this changes? What the benefits , anything to watch out for, has QA test and verify?

Reply:
Using AV1/VP9 as default codec will provide better video compression efficiency than H.264 and improve video quality. While the two codec may be more costly in computation, we already added monitoring for encoding and fallback when there are failures or high CPU usage. Several major customers have successfully switched to the configuration to use VP9 as default codec for video and AV1 for content. QA has tested the change and verified.

Reply:
Sorry i mean if we can document it in CHANGELOG on why the changes are needed and PR description on more information on testing :D

Reply:
Updated CHANGELOG and PR description